app/javascript/article-form/components/TagsField.jsx

```diff
@@ -29,20 +29,19 @@ export const TagsField = ({ onInput, defaultValue, switchHelpContext }) => {
     // Fetching further tag data allows us to display a richer UI
     // This fetch only happens once on first component load
     if (defaultValue && defaultValue !== '' && !defaultsLoaded) {
-      const tagNames = defaultValue.split(', ');
+      const tagNamesQueryString = defaultValue
+        .split(', ')
+        .reduce((queryString, nextTagName) => {
```

Comment:
I don't think it's the case as we weren't doing it before, but do tag names need to be encoded for the query string or do all the allowable characters for a tag not require encoding?

Reply:
That's a good question! I double-checked the `Tag` model just now and we only create tags with names that contain alphanumeric characters so I think we're safe 👍 

https://github.com/forem/forem/blob/main/app/models/tag.rb#L121-L127